src/pages/ApplicationsPage.tsx

```diff
@@ -1,23 +1,59 @@
 import React from 'react';
 import { Helmet } from 'react-helmet';
 import { useFeatureFlag } from '@openshift/dynamic-plugin-sdk';
-import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { Bullseye, PageSection, PageSectionVariants, Spinner } from '@patternfly/react-core';
 import ApplicationDetailsView from '../components/ApplicationDetailsView/ApplicationDetailsView';
 import ApplicationListView from '../components/ApplicationListView/ApplicationListView';
+import { GettingStartedCard } from '../components/GettingStartedCard/GettingStartedCard';
+import { GettingStartedModal } from '../components/modal/GettingStartedModal';
 import NamespacedPage from '../components/NamespacedPage/NamespacedPage';
 import PageLayout from '../components/PageLayout/PageLayout';
 import HacbsApplicationDetails from '../hacbs/components/ApplicationDetails/HacbsApplicationDetails';
 import { HACBS_FLAG } from '../hacbs/hacbsFeatureFlag';
+import { useActiveNamespace } from '../hooks';
 import { useQuickstartCloseOnUnmount } from '../hooks/useQuickstartCloseOnUnmount';
+import imageUrl from '../imgs/getting-started-illustration.svg';
+import { ApplicationGroupVersionKind } from '../models';
 import { getQueryArgument } from '../shared/utils';
+import { ApplicationKind } from '../types';
+
+const GETTING_STARTED_MODAL_KEY = 'application-list-getting-started-modal';
+const GETTING_STARTED_CARD_KEY = 'application-list-getting-started-card';
 
 const ApplicationsPage = () => {
   useQuickstartCloseOnUnmount();
   const applicationName = getQueryArgument('name');
   const [hacbs] = useFeatureFlag(HACBS_FLAG);
 
+  const [namespace, namespaceLoaded] = useActiveNamespace();
```

Comment:
How is this going to work? The namespace context is getting set in this component only.

Reply:
It would also mean that the first time the page is loaded, namespace will be empty string and the resource fetch hook would make a call with `''` as namespace.